### PR TITLE
Added licencse templates

### PR DESCRIPTION
**Pull request by mustermeiszer**
Here are the templates for the file headers.

### CODE REVIEW

File: apache-template.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```
Comment (vedhavyas) [inline apache-template.rs:1]: do we need them to be a rust file? can we do away with simple text file?

**Review by vedhavyas — COMMENTED** [review]
I think the we need to add apache license as well. It is missing at the moment.

File: apache-template.rs (hunk 1 above)
Comment (mustermeiszer) [reply apache-template.rs:1]: No need to be rust. Can be txt. Will change this.

**Review by vedhavyas — APPROVED** [review]
LGTM

**Pull request merged**